cmake/Conan.cmake

```diff
@@ -1,33 +1,44 @@
 macro(run_conan)
   # Download automatically, you can also just copy the conan.cmake file
   if(NOT EXISTS "${CMAKE_BINARY_DIR}/conan.cmake")
-    message(STATUS "Downloading conan.cmake from https://github.com/conan-io/cmake-conan")
-    file(DOWNLOAD "https://github.com/conan-io/cmake-conan/raw/v0.16.1/conan.cmake" "${CMAKE_BINARY_DIR}/conan.cmake")
+    message(
+      STATUS
+        "Downloading conan.cmake from https://github.com/conan-io/cmake-conan")
+    file(
+      DOWNLOAD
+      "https://raw.githubusercontent.com/conan-io/cmake-conan/v0.16.1/conan.cmake"
+      "${CMAKE_BINARY_DIR}/conan.cmake"
+      EXPECTED_HASH
+        SHA256=396e16d0f5eabdc6a14afddbcfff62a54a7ee75c6da23f32f7a31bc85db23484
+      TLS_VERIFY ON)
   endif()
 
-  set (ENV{CONAN_REVISIONS_ENABLED} 1)
+  set(ENV{CONAN_REVISIONS_ENABLED} 1)
+  list(APPEND CMAKE_MODULE_PATH ${CMAKE_BINARY_DIR})
+  list(APPEND CMAKE_PREFIX_PATH ${CMAKE_BINARY_DIR})
+
   include(${CMAKE_BINARY_DIR}/conan.cmake)
 
-  conan_add_remote(
-    NAME
-    conan-center
-    URL
-    https://api.bintray.com/conan/conan/conan-center)
+  # Add (or remove) remotes as needed
+  # conan_add_remote(NAME conan-center URL https://conan.bintray.com)
+  conan_add_remote(NAME cci URL https://center.conan.io INDEX 0)
 
-  conan_add_remote(
-    NAME
-    bincrafters
-    URL
-    https://bincrafters.jfrog.io/artifactory/api/conan/public-conan)
```

Comment:
I am unclear as to why this remote was removed. I understand the concept of "Add or remove remotes as needed", but this remote is absolutely required if you want to get the SDL example running. Perhaps this was done unintentionally?

Reply:
Additionally, in case anyone else is trying to get the SDL example to build, I'd like to share that you will need to run it using a Python before 3.9. On Linux, SDL's dependency tree includes mesa, and some of the Python code that builds mesa relies on some crufty XML parser code that was deprecated many years ago, and was removed in Python 3.9.